athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/manager/JDBCUtil.java

```diff
@@ -170,6 +198,24 @@ public static PreparedStatement getTableNameQuery(Connection connection, String
         return preparedStatement;
     }
 
+    public static PreparedStatement getSchemaNameQuery(Connection connection, String databaseName) throws SQLException
+    {
+        String sql = "SELECT schema_name FROM information_schema.schemata WHERE (schema_name = ? or lower(schema_name) = ?)";
+        PreparedStatement preparedStatement = connection.prepareStatement(sql);
+        preparedStatement.setString(1, databaseName);
+        preparedStatement.setString(2, databaseName);
+        return preparedStatement;
+    }
+
+    public static PreparedStatement getRedshiftSchemaNameQuery(Connection connection, String databaseName) throws SQLException
```

Comment:
Other team members can weigh in, but I think it would make more sense to separate helper functions specific to Redshift, into a separate Util class.

Reply:
+1